client/lib/directly/test/index.js

```diff
@@ -38,8 +34,8 @@ describe( 'index', () => {
 		if ( script ) {
 			script.remove();
 		}
-		delete window.DirectlyRTM;
-		delete require.cache[ require.resolve( '..' ) ];
+		window.DirectlyRTM = undefined;
+		jest.resetModules();
```

Comment:
We should include extra documentation for when it makes sense to use `resetModules`.